css/index.css

```diff
@@ -143,4 +143,288 @@ footer nav {
+    flex-direction: column;
+	margin-top: 30px;
+	margin-bottom: 25px;
+	padding-bottom: 30px;
+    border-bottom: 1px dotted black;
+    justify-content: center;
+    margin-left: 20px;
+}
+
+.about-middle-content {
+    display: flex;
+	flex-wrap: wrap;
+    justify-content: space-between
+}
+
+.about-middle-content div {
+    display: flex;
+    width: 45%;
+    margin-bottom: 40px;
+    padding-bottom: 20px;
+	flex-wrap: wrap;
+	padding-left: 20px;
+    margin-top: 10px;
+    border-bottom: 1px dotted black;
+}
+
+.container .about-together {
+    margin-bottom: 40px;
+    margin-left: 20px;
+}
+
+.about-middle-content h2 {
+    margin-top: 15px;
+}
+
+.about-middle-content span {
+	margin-top: 20px;
+	padding: 12px 40px;
+	border: 2px solid black;
+	font-size: 13px;
+	border-radius: 10px;
+}
+
+.button {
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19);
+}
+
+/* Blog */
+.top-content-blog {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    width:100%;
+    text-align: center;
+    margin: 10% 0;
+    min-height: 200px;
+    background-color: teal;
+    align-items: center;
+    font-style:bold;
+    border-radius: 25px;
+  color:white;
+  
+} 
+
+.bloglogo {
+    width: 100%;
+    height: auto;
+  }
+
+/* contact */
+.contact-us h2 {
+    font-size: 20px;
+  }
+  .contact-top {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    margin: 2rem 0;
+  }
+  
+  .contact-form {
+    border-radius: 5px;
+    box-shadow: 0px 0px 10px 0px rgba(0, 0, 0, 0.75);
+    padding: 0.5rem;
+    display: flex;
+    flex-direction: column;
+    margin: 1.5rem 0;
+  }
+  
+  .form-contact-info {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 0.5rem;
+  }
+  
+  .contact-form a {
+    border: 2px solid black;
+    border-radius: 10px;
+    margin: 1rem auto;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-decoration: none;
+    color: black;
+    height: 2rem;
+    width: 50%;
+  }
+
+   /* Style inputs with type="text", select elements and textareas */
+input[type=text], select, textarea {
+    width: 100%; /* Full width */
+    padding: 12px; /* Some padding */ 
+    border: 1px solid #ccc; /* Gray border */
+    border-radius: 4px; /* Rounded borders */
+    box-sizing: border-box; /* Make sure that padding and width stays in place */
+    margin-top: 6px; /* Add a top margin */
+    margin-bottom: 16px; /* Bottom margin */
+    resize: vertical /* Allow the user to vertically resize the textarea (not horizontally) */
+  }
+  
+  /* Style the submit button with a specific background color etc */
+  input[type=submit] {
+    background-color: #4CAF50;
+    color: white;
+    padding: 12px 20px;
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+  }
+  
+  /* When moving the mouse over the submit button, add a darker green color */
+  input[type=submit]:hover {
```

Comment:
I see indications everywhere, such as here, that indicate your deep understanding of CSS. Great work!